examples/lambda/wcbv-reasonable/LScript.sml

```diff
@@ -0,0 +1,576 @@
+(* See:
```

Comment:
Please give this theory a better name than just `L`

Reply:
Might use the name `WCBVLC` or `WCBV` instead of `L`.

Reply:
Or `weak_CBV`?

Reply:
`weak_CBV` sounds good.

Reply:
You need to Holmake either `cleanDeps` or `cleanAll` to purge old files that still think `LScript` is a thing.